### DISCUSSION TITLE
How to ignore `--app-dir`?

### DISCUSSION THREAD

**Discussion by IlIllIl** [Homebrew/discussions#5783] [Writing Formulae/Casks]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

Some applications must be installed in the `/Applications` directory otherwise they will not work properly. Some of them can move themselves, but some require manual operation, which may cause `brew upgrade` issues. So is there a way to add specific options for them in `cask.rb`?

**Comment by p-linnane** [comment]
https://docs.brew.sh/Manpage#global-cask-options

**Comment by cho-m** [comment]
Also see https://github.com/Homebrew/homebrew-cask/issues/12858. Pull requests are welcome to add a DSL to support this.
